.github/workflows/build.yml

```diff
@@ -63,9 +63,10 @@ jobs:
       with:
         usesh: true
         run: |
-          ./configure
-          make
-          make roundtrip
+          pkg install -y llvm gmake
+          MAKE=gmake ./configure
```

Comment:
My comment [from issue thread](https://github.com/kspalaiologos/bzip3/issues/22#issuecomment-1126725593):

> I think forcing `gmake` was a mistake, we *should* be testing with BSD's native make and we should keep it fully portable so that every platform's default `make` works out of the box. Why even bother testing on BSD in we just force GNU tooling on it? That will not work out well when more obscure *nixes want to build that make not even have that as an option.